rfcs/52-unstable_features.md

```diff
@@ -0,0 +1,84 @@
+# Feature Name: `unstable_features`
+
+## Summary
+
+Accepting PRs for new features that may not have reach the high bar we set for Bevy, and disable their changes with a feature gate.
+
+## Motivation
+
+Some features would benefit from iterative work in Bevy, but that may be hard to do when a PRs need a high level of quality to be merged in.
+
+In this RFC, "feature" means the new feature or change under development, and "feature gate" the conditional compilation flag. "unstable" means a feature that is subject to change, not something that will break Bevy.
+
+## User-facing explanation
+
+So you want to contribute a new feature to Bevy! You have a few paths available to you, depending on the scope of the feature:
+- A small feature with a clear implementation? Go ahead and open that PR!
+- A large feature with an open design space that could use some discussion about how to implement it? A RFC would be good.
+- Something in between that would benefit from iterative work? Here come the unstable feature workflow for you!
+
+You will need to open a tracking issue to explain the new feature you want to add. If it looks good, you'll be ready to open that PR with a feature gate for your changes!
+
+## Implementation strategy
+
+The new workflow for this kind of feature would be:
+- Submit an issue explaining the feature you want to add under a feature gate. This will be the tracking issue for stabilization once approved. This issue gets a new label `S-Stabilization`.
+- If approved, submit a PR with the new feature and the feature gate
+- This PR can be merged by someone with the merge rights on the related part of Bevy as soon as it's `S-Ready-For-Final-Review`, without an approval by the main Bevy maintainers.
```

Comment:
We should further expand this. Currently there are three people with merge rights, and only one of them is unscoped. This, at best, seems to still have a heavy bottleneck here. We already have engine teams for specific focus areas. Would it be possible to set it up such that areas of the engine could have these unstable features merged with significant focus area engine team buy-in? The only other way is to expand the number of org members with scoped merge rights, and that seems to be the slowest area of growth for the organization as a whole.

Reply:
I think that is out of scope of this RFC. I wrote it mentioning roles, who has those roles is not the subject. What you propose would still work in the context of this RFC, as it can be read as "giving the focus area teams the scope to merge approved unstable features PRs".

Reply:
Can we mark this as resolved now that all maintainers have merge rights?